packages/template-retail-react-app/config/default.js

```diff
@@ -9,6 +9,7 @@ const sites = require('./sites.js')
 
 module.exports = {
     app: {
+        commerceAgent: process.env.COMMERCE_AGENT_SETTINGS || JSON.stringify({enabled: 'false'}),
```

Comment:
```suggestion
        commerceAgent: process.env.COMMERCE_AGENT_SETTINGS || JSON.stringify({enabled: false}),
```

Reply:
Follow up comment. Why are we maintaining this as a string vs an object?

```suggestion
        commerceAgent: JSON.parse(process.env.COMMERCE_AGENT_SETTINGS || {enabled: 'false'}),
```